tests/functional/dependencies/test_dependency_options.py

```diff
@@ -34,9 +34,11 @@ def test_deps_lock(self, clean_start):
         with open("package-lock.yml") as fp:
             contents = fp.read()
 
-        fivetran_package = "- package: fivetran/fivetran_utils\n    version: 0.4.7"
+        fivetran_package = (
```

Comment:
nit: I wonder if this would be clearer to read with a heredoc.

Reply:
qq chose comma ça ?
```python
        fivetran_package = """
  - name: fivetran_utils
    package: fivetran/fivetran_utils
    version: 0.4.7
"""

Reply:
If you also feel that it increases comprehension, I'm 50/50.

Reply:
I do think it's easier to read!